lino/modlib/extjs/views.py

```diff
@@ -608,12 +606,12 @@ def get(self, request, app_label=None, actor=None):
             total_count = ar.get_total_count()
             # raise Exception("20171208 {}".format(ar.data_iterator.query))
             for row in ar.create_phantom_rows():
-                if len(rows)+1 < ar.limit\
-                        or ar.limit == total_count + 1:
+                if (ar.limit is not None and len(rows) + 1 < ar.limit) or (
+                                ar.limit is None and isinstance(row, PhantomRow)) or ar.limit == total_count + 1:
```

Comment:
Add a check if the limit is None and the row is a PhantomRow so we actually append the row.